content/docs/for-platform-operators/concepts/build-run-images.md

```diff
@@ -0,0 +1,94 @@
++++
+title="Base image types"
+weight=99
++++
+
+As you already know, `Cloud Native Buildpacks (CNBs)` transform your application source code into `OCI images`  that can run on any cloud and include all of the components your application needs to build and run and.
+
+<!--more-->
+
+Each buildpack checks the source code and provides any relevant dependencies. Then, a `base image` is then generated from the application’s source code and these dependencies.
+
+## Base image types
+
+A `base image` is an `OCI image` containing the base, or initial set of layers, for other images. It is helpful to distinguish between two distinct types of images, `Build` and `Run` images.
+
+### Build image
+
+A `build image` is an `OCI image` that serves as the base image for the `build` environment in which buildpacks `lifecycle` is executed.
+
+A typical `build image` might determine:
+
+* The OS distro in the build environment
+* OS packages installed in the build environment
+* Trusted CA certificates in the build environment
+* The default user in the build environment
+
+The platform must ensure that:
+
+* The image config's `User` field is set to a non-root user with a writable home directory
+* The image config's `Env` field has the environment variable `CNB_USER_ID` set to the user [UID/SID](https://github.com/buildpacks/spec/blob/main/README.md#operating-system-conventions) of the user specified in the `User` field
```

Comment:
We may have to explain what is meant by "`User` field" and "`Env` field".  Do we mean user and environment here?  i.e. that the build image must have a non-root user and `CNB_USER_ID` is an environment variable which contains the `uid` of that user.

Reply:
In general, I think we should avoid copy/pasting the spec into the docs. Then we just have two things to maintain. This section should provide some grounding for understanding the spec, and then link to it. Let's think about how to frame this and then provide a code suggestion.

Reply:
@natalieparellano then we would need something more broad that we could use. The spec is very specific and I can't change things as every line is very focused!! Do we have any other resources that I could use?

Reply:
We don't have other sources.  We really struggle to explain these things.  Let's chat about this in our regular catchup.  I think there's a straightforward solution here.  We may have to unpack the idea of an _image config_ using an example, so that readers get a clear idea of what these fields look like.